share/gpodder/extensions/sonos.py

```diff
@@ -25,38 +25,35 @@
 
 SONOS_CAN_PLAY = lambda e: 'audio' in e.file_type()
 
-
```

Comment:
keep 2 blank lines between top level items (pycodestyle)

Reply:
This must have been a mis-paste.